itomIsoGLFigurePlugin/plotIsoGLWidget.h

```diff
@@ -136,8 +136,8 @@ class plotGLWidget : public QGLWidget
         bool lightArrowEnabled();
         void paintGL();
         void resizeGL(int width, int height);
-//        void paintEvent(QPaintEvent *pevent);
-//        void resizeEvent(QResizeEvent *pevent);
+//        void paintEvent(QPaintEvent *prevent);
```

Comment:
do not change, keep ```pevent```. Same for the next line. To avoid changes in the next run of this hook either rename the variable or set ```pevent``` to the whitelist.